deployment/local/simple_server.py

```
@@ -0,0 +1,84 @@
+def static_files(filename):
+    website_dir = os.path.join(os.path.dirname(__file__), '..', '..', 'website')
+    return send_from_directory(website_dir, filename)
+
+# CORS Proxy for Real API
+import requests
+
+@app.route('/api/emotion', methods=['POST'])
+def proxy_emotion():
+    try:
+        # Get text from query parameter
+        text = request.args.get('text', '')
```

Ruff (0.12.2)

Undefined name `request`

(F821)